doc/2/controllers/auth/check-rights/snippets/check-rights.dart

```diff
@@ -0,0 +1,13 @@
+final requestPayload = {
+  'controller': 'document',
+  'action': 'create',
+  'index': 'nyc-open-data',
+  'collection': 'yellow-taxi',
+  'body': {
+    'name': 'Melis'
+  }
+};
+
+final result = await kuzzle
```

Comment:
```suggestion
final allowed = await kuzzle
```

Reply:
It must be named `result` as the template is printing it.